packaging_automation/tests/test_citus_package.py

```diff
@@ -16,6 +16,7 @@
 PACKAGING_SOURCE_FOLDER = "packaging_test"
 PACKAGING_EXEC_FOLDER = f"{TEST_BASE_PATH}/{PACKAGING_SOURCE_FOLDER}"
 BASE_OUTPUT_FOLDER = f"{PACKAGING_EXEC_FOLDER}/packages"
+CURRENT_BRANCH = "all-citus"
 MAIN_BRANCH = "all-citus"
```

Comment:
I think we can remove those two global variables and just set them in `test_upload_to_package_cloud` since they don't seem to be referenced from elsewhere

Reply:
Fixed